install/kubernetes/helm/istio/charts/tracing/templates/service.yaml

```diff
@@ -33,9 +33,9 @@ items:
   spec:
     ports:
       - name: query-http
-        port: 80
+        port: {{ .Values.service.externalUiPort }}
         protocol: TCP
-        targetPort: {{ .Values.service.uiPort }}
+        targetPort: {{ .Values.service.internalUiPort }}
```

Comment:
I would like to avoid proliferation and inconsistent use of nodePort in all services.

Can you use the gateway ? Ideally just gateway should be exposed via nodeport or LB, other services should use the istio APIs.